Sources/BugsnagPerformance/Private/BugsnagPerformanceLibrary.mm

```diff
@@ -67,6 +68,7 @@
         }
 
         sharedInstance().configure(config);
+        BugsnagPerformanceCrossTalkAPI.sharedInstance.configuration = config;
```

Comment:
Instead of this, let's convert `BugsnagPerformanceCrossTalkAPI` to implement `PhasedStartup`. Then `config` becomes a private member, and we don't have to worry about what's being externally filled in from where.